WebRoot/js/zimbraMail/calendar/controller/ZmCalViewController.js

```diff
@@ -259,7 +259,8 @@ function(viewId, startDate, skipMaintenance) {
             break;
     }
     if (viewId == ZmId.VIEW_CAL_LIST) {
-		this._navToolBar[ZmId.VIEW_CAL].setVisible(false);
+		var navText = currentView.getCalTitle();
```

Comment:
else block is also having similar statement for setting the title, can we move it out of if/else block?

Reply:
or can we use view._updateTitle method?

Reply:
Trying to merge the common code. Can't use the view._updateTitle because we trying to get the title & update navbar date range value. The updateTitle is meant to update the model.

Reply:
Moved common code out of if block here:
defb1cc61974d33ac5fba8f1e0c4249a49993de6